### PR TITLE
Feature/accounts UI fix

### PR DESCRIPTION
**Pull request by MarcusNkadimeng**
This is my UI fix not just Accounts screen UI fix. Please do not mind the name. Also it is a lot of files but only because I was touching the storyboards and the nibs.

Merge - feature/Accounts-UI-fix -> Dev
 
 ## Description 
UI fixes for entire application
 
## Screenshots/GIFs 
Screen Recording 2024-05-28 at 16 05 01
 Screenshot 2024-05-28 at 16 04 53
Simulator Screenshot - iPhone 15 Pro - 2024-05-28 at 16 12 19

## Checklist 
Please review and check the following before submitting this pull request: 
- [x] The code follows the project's coding standards. 
- [x] 100% Unit test coverage for View Models 
- [ ] All warnings Resolved.

### CODE REVIEW

File: budgetApp1/budgetApp1/AccountViewCell.swift [hunk 1]
```diff
@@ -48,9 +45,9 @@ class AccountViewCell: UITableViewCell {
         containerView.layer.cornerRadius = 10.0
         containerView.layer.masksToBounds = true
         containerView.layer.borderWidth = 2.0
-        containerView.layer.borderColor = uiSpecs.tetiaryColour.cgColor
+        containerView.layer.borderColor = UIColor.accent.cgColor
```
Comment (kieran-woodrow) [inline budgetApp1/budgetApp1/AccountViewCell.swift:48]: is this one of the asset colours?

File: budgetApp1/budgetApp1/LoginViewController.swift [hunk 2]
```diff
@@ -30,4 +30,34 @@ class LoginViewController: UIViewController {
...
+        username.layer.borderColor = UIColor.accent.cgColor
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(keyboardWillShow),
+                                               name: UIResponder.keyboardWillShowNotification,
+                                               object: nil)
+        NotificationCenter.default.addObserver(self,
+                                               selector: #selector(keyboardWillHide),
+                                               name: UIResponder.keyboardWillHideNotification,
+                                               object: nil)
+    }
```
Comment (kieran-woodrow) [inline budgetApp1/budgetApp1/LoginViewController.swift:49]: new line

File: budgetApp1/budgetApp1/LoginViewController.swift [hunk 3]
```diff
@@ -30,4 +30,34 @@ class LoginViewController: UIViewController {
...
+    @objc private func keyboardWillShow(notification: NSNotification) {
+        if let keyboardSize = (notification.userInfo?[UIResponder.keyboardFrameEndUserInfoKey] as? NSValue)?.cgRectValue {
+            if self.view.frame.origin.y == 0 {
+                self.view.frame.origin.y -= 150
+            }
+        }
+    }
```
Comment (kieran-woodrow) [inline budgetApp1/budgetApp1/LoginViewController.swift:57]: nerw line

**Review by kieran-woodrow — DISMISSED** [review]

File: budgetApp1/budgetApp1/AccountViewCell.swift (hunk 1 above)
Comment (MarcusNkadimeng) [reply budgetApp1/budgetApp1/AccountViewCell.swift:48]: yes

**Review by kieran-woodrow — APPROVED** [review]

**Review by giacatano — APPROVED** [review]

**Pull request merged**